src/components/Popover/index.tsx

```diff
@@ -26,6 +26,7 @@ const Arrow = styled.div`
     position: absolute;
     width: 8px;
     height: 8px;
+    box-sizing: border-box;
```

Comment:
Much easier to reason about, otherwise we would have a 9x9 box (because it has two borders) inside a 8x8 box.